linera-base/src/unit_tests.rs

```diff
@@ -47,6 +48,36 @@ where
         .expect("Flattening WIT roundtrip test failed");
 }
 
+/// Test GraphQL serialization roundtrip.
+#[test_case(account_test_case(); "of_account")]
```

Comment:
Are there more types that could use these type of tests? Maybe we can enforce it? i.e. for example when we generate `*.graphql` schemas we could run these tests over all types that are outputted there.